pcsx2/gui/MainFrame.cpp

```diff
@@ -437,13 +437,15 @@ MainEmuFrame::MainEmuFrame(wxWindow* parent, const wxString& title)
 	m_menuSys.Append(MenuId_EnableWideScreenPatches,	_("Enable &Widescreen Patches"),
 		wxEmptyString, wxITEM_CHECK);
 
-	m_menuSys.Append(MenuId_EnableHostFs,	_("Enable &Host Filesystem"),
-		wxEmptyString, wxITEM_CHECK);
+	if(IsDebugBuild || IsDevBuild)
+		m_menuSys.Append(MenuId_EnableHostFs,	_("Enable &Host Filesystem"),
+			wxEmptyString, wxITEM_CHECK);
 
 	m_menuSys.AppendSeparator();
 
 	m_menuSys.Append(MenuId_Sys_Shutdown,	_("Shut&down"),
 		_("Wipes all internal VM states and shuts down plugins."));
+	m_menuSys.FindItem(MenuId_Sys_Shutdown)->Enable(false);
```

Comment:
So I know it's not great to run external programs with PCSX2; but I've ran FRAPS with PCSX2 all the time (as an FPS meter) and I've never had any issues until now.  SuperVU fails to allocate memory if the First Time Wizard is run while a FRAPS process is active. After running the First Time Wizard, the plugins are shutdown right before virtual memory mapping occurs (if you reopen the .exe, this doesn't happen)
	```Shutdown DEV9 and so on``` ...
```Plugins shutdown successfully```
```Reserving memory for recompilers...```
```SuperVU0 Recompiler Cache: host memory @ 0x0E800000 -> 0x0F000000 is unavailable; attempting to map elsewhere...```
```SuperVU1 Recompiler Cache: host memory @ 0x0F000000 -> 0x0F800000 is unavailable; attempting to map elsewhere...```
	```CPU provider error:```
	```Virtual memory map failed while reserving 'SuperVU1 Recompiler Cache':```
	```SuperVU failed to allocate virtual memory below 256MB.```

Do you think this is related to any changes in this PR? I ran the FTW with an older build/FRAPS running; and had no problems with the sVU rec. It would be good to find out if other programs cause this as well, so we don't have any issues in the upcoming stable release.